submissions/xT0PERx/Memory pair game/transitions.js

```diff
@@ -0,0 +1,139 @@
+const imageses = [
+  'images/Alfa-romeo.jpeg',
+  'images/Ford.jpeg',
+  'images/Mazda.jpeg',
+  'images/Nissan.jpeg',
+  'images/Peugeot.jpeg',
+];
+
+const cardDeck = [...imageses, ...imageses];
+
+function getingSortList() {
+  cardDeck.sort(function () {
+    return 0.5 - Math.random();
+  });
+}
+
+let winCount = 0;
+
+//this func must added imgs in all my cards
+const allCardsOnTheDesk = document.querySelector('main');
+const putImgsInCards = function () {
+  const fragment = document
+    .createDocumentFragment()
+    .appendChild(allCardsOnTheDesk);
+
+  for (let i = 0; i < cardDeck.length; i++) {
+    const img = document.createElement('img');
+    img.src = cardDeck[i];
+    fragment.children[i].appendChild(img);
+  }
+  document.body.appendChild(fragment);
+};
+
+// const overturnsCard = ({ target }) => {
+//   if (target.nodeName !== 'DIV') {
+//     return;
+//   }
+
+//   if (target.classList.contains('card__clicked') === true) {
+//     target.classList.remove('card__clicked');
+//     target.classList.add('card');
+//   } else if (target.classList.contains('card') === true) {
+//     target.classList.remove('card');
+//     target.classList.add('card__clicked');
+//   }
+//   compareImgs();
+// };
+
+// allCardsOnTheDesk.addEventListener('click', overturnsCard());
+
+const overturnsCard = function () {
+  allCardsOnTheDesk.addEventListener('click', function ({ target }) {
+    if (target.nodeName !== 'DIV') {
+      return;
+    }
+
+    if (target.classList.contains('card__clicked')) {
+      target.classList.remove('card__clicked');
```

Comment:
In the description of the task, a link was attached with an example of flipping cards
Try using an [example](https://davidwalsh.name/css-flip)